File: Data/Models/AccessUser.cs
namespace Blackwood.Access.Models
{
    using System;
    using System.ComponentModel.DataAnnotations;

    public class AccessUser
    {
        [Key]
        public int Id { get; set; }
        public string DomainUsername { get; set; }
        public string AccountName { get; set; }
        public bool IsActive { get; set; } = true;
        public bool IsAdmin { get; set; } = false;
        public bool IsPayrollUser { get; set; } = false;
        public int DefaultTeamCode { get; set; } = 0;
        public bool CanAuthoriseAdjustments { get; set; } = false;
        public bool CanRejectAdjustments { get; set; } = false;
        public bool IsAssessmentUser { get; set; } = false;
    }
}

File: ClientApp/app/models/AccessUser.ts
import { AccessUserTeam } from "./AccessUserTeam";

export class AccessUser {
    public id: number = 0;
    public accountName: string;
    public isActive: boolean = true;
    public isAdmin: boolean;
    public isPayrollUser: boolean;
    public defaultTeamCode: number;
    public canAuthoriseAdjustments: boolean;
    public canRejectAdjustments: boolean;
    public isAssessmentUser: boolean;
    public isReportingUser: boolean;
    public isAccidentUser: boolean;
    public isIntegrationUser: boolean;

    public emailAddress: string;
    public domainUsername: string;

    public authorizedTeams: AccessUserTeam[] = [];
}